frontend/src/components/OAuth2Session.test.tsx

```diff
@@ -48,6 +48,7 @@ describe("<OAuth2Session />", () => {
 
   const finishedAt = "2023-06-29T03:35:19.451292+00:00";
 
+  beforeAll(async () => await routerReady);
```

Comment:
This can probably live in `test-utils/router` - so merely importing it would do it - if that isn't too magical for you

Reply:
You're right that works. Just did that